elrond-node.md

```diff
@@ -159,20 +159,20 @@ The `<callStack>` cell stores a list of previous contract execution states. Thes
     rule <commands> pushCallState => . ... </commands>
          <callStack> (.List => ListItem(CALLSTATE)) ... </callStack>
          <callState> CALLSTATE </callState>
-      [priority(60)]
+      [priority(10)]
```

Comment:
They need to have a higher priority than wasm instructions because switching the context (pushing/popping the callstate + worldstate) is 2 steps and the instructions should not execute in the middle of it. 

For example there was a situation like this IIRC, when returning from a failed contract call, `#exception ~> #endWasm` command rewrites to `#popCallState ~> #popWorldState`. After the `#popCallState` we get a `#finishExOnDestCtx` in `<instrs>`, which will be executed before `#popWorldState`.

Reply:
I decided not to change the priorities because It could break other things.